stripe/src/main/java/com/stripe/android/util/StripeTextUtils.java

```diff
@@ -13,26 +32,72 @@ public static boolean hasAnyPrefix(String number, String... prefixes) {
         return false;
     }
 
+    /**
+     * Check to see whether the input string is a whole, positive number.
+     *
+     * @param value the input string to test
+     * @return {@code true} if the input value consists entirely of integers
+     */
     public static boolean isWholePositiveNumber(String value) {
         if (value == null) {
             return false;
         }
-        for (char c : value.toCharArray()) {
-            if (!Character.isDigit(c)) {
-                return false;
-            }
-        }
-        return true;
+
+        return TextUtils.isDigitsOnly(value);
     }
 
+    /**
+     * Swap {@code null} for blank text values.
+     *
+     * @param value an input string that may or may not be entirely whitespace
+     * @return {@code null} if the string is entirely whitespace, or the original value if not
+     */
     public static String nullIfBlank(String value) {
         if (isBlank(value)) {
             return null;
         }
         return value;
     }
 
+    /**
+     * A checker for whether or not the input value is entirely whitespace. This is slightly more
+     * aggressive than {@link TextUtils#isEmpty(CharSequence)}, which only returns true for
+     * {@code null} or {@code ""}.
+     *
+     * @param value a possibly blank input string value
+     * @return {@code true} if and only if the value is all whitespace, {@code null}, or empty
+     */
     public static boolean isBlank(String value) {
         return value == null || value.trim().length() == 0;
     }
+
+    /**
+     * Converts an unchecked String value to a {@link CardBrand} or {@code null}.
+     *
+     * @param possibleCardType a String that might match a {@link CardBrand} or be empty.
+     * @return {@code null} if the input is blank, else the appropriate {@link CardBrand}.
+     */
+    @Nullable
+    @CardBrand
+    public static String asCardBrand(@Nullable String possibleCardType) {
+        if (isBlank(possibleCardType)) {
+            return null;
+        }
+
+        if (Card.AMERICAN_EXPRESS.equals(possibleCardType)) {
```

Comment:
Also realized this method 1) should ignore case because Javascript will do that when passing things back and forth from the server, and 2) should have unit test coverage.